internal/web3ext/web3ext.go

```diff
@@ -28,8 +28,36 @@ var Modules = map[string]string{
 	"rpc":        RPC_JS,
 	"shh":        Shh_JS,
 	"txpool":     TxPool_JS,
+	"swarmfs":    SWARMFS_JS,
 }
 
+const SWARMFS_JS = `
```

Comment:
Please keep the list is this file sorted alphabetically. Both the above and below too (swarm is notorious for adding stuff all over the place in this file :P)